projects/example-app/src/README.md

```diff
@@ -12,15 +12,15 @@ Built with [@angular/cli](https://github.com/angular/angular-cli)
 
 ### Included
 
-* [@ngrx/store](../docs/store/README.md) - RxJS powered state management for Angular apps, inspired by Redux
-* [@ngrx/effects](../docs/effects/README.md) - Side effect model for @ngrx/store
-* [@ngrx/router-store](../docs/router-store/README.md) - Bindings to connect the Angular Router to @ngrx/store
-* [@ngrx/entity](../docs/entity/README.md) - Entity State adapter for managing record collections.
-* [@ngrx/store-devtools](../docs/store-devtools/README.md) - Instrumentation for @ngrx/store enabling time-travel debugging
-* [@ngrx/db](https://github.com/ngrx/db) - RxJS powered IndexedDB for Angular apps
-* [@angular/router](https://github.com/angular/angular) - Angular Router
-* [@angular/material](https://github.com/angular/material2) - Angular Material
-* [jest](https://facebook.github.io/jest/) - JavaScript test runner with easy setup, isolated browser testing and snapshot testing
+- [@ngrx/store](../docs/store/README.md) - RxJS powered state management for Angular apps, inspired by Redux
```

Comment:
these paths (store, effects, router-store, entity, devtools) should go two levels higher, likewise for the same paths at the start of the README.